genexuslabs/web-controls-library
# Update Stencil and fix vulnerabilities reported by npm audit

#### Changes proposed in this Pull Request

Update Stencil and fix security vulnerabilities reported by npm audit.
The security vulnerabilities are always related to the same outdated and unmaintained packages, used to generate documentation for SASS mixins for theming.
Until we find a better alternative, we will remove these packages and stop auto-generating those docs.

#### Testing instructions

Checkout this branch.
Run `npm run validate` without errors.
Run `npm audit` and check that there are no vulnerabilities.
